README.md

```diff
@@ -54,40 +56,61 @@ Write-AudioDevice -PlaybackMeter  # Writes the default playback device's power o
 
 ## Build Cmdlet from source
 
-1. Using Visual Studio Community, create new project from SOURCE folder  
-    File -> New -> Project From Existing Code...
-    
+1. Install Visual Studio 2022
+
+		Workloads: .NET desktop development
+
+2. Create new project from SOURCE folder  
+File -> New -> Project From Existing Code...
+
 		Type of project: Visual C#
 		Folder: SOURCE
 		Name: AudioDeviceCmdlets
 		Output type: Class Library
 
-2. Install System.Management.Automation NuGet package  
+2. Install System.Management.Automation NuGet legacy package, which is packaged as part of Microsoft.PowerShell.5.1.ReferenceAssemblies  
     Project -> Manage NuGet Packages...
 
-		Browse: System.Management.Automation
-		Install: v6.3+
+		Browse: Microsoft.PowerShell.5.1.ReferenceAssemblies
+		Install: v1.0.0+
 
 3. Set project properties  
-	Project -> AudioDeviceCmdlets Properties...
+Project -> AudioDeviceCmdlets Properties
 
 		Assembly name: AudioDeviceCmdlets
-		Target framework: .NET Framework 4.5+
+		Target framework: .NET Framework 4.6.1+
 
-4. Set solution configuration  
-    Build -> Configuration Manager...
+4. Install System.Management.Automation NuGet legacy package  
+Project -> Manage NuGet Packages...
+
+		Package source: nuget.org
+		Browse: Microsoft.PowerShell.5.1.ReferenceAssemblies
+		Install: v1.0.0+
+
+5. Set solution configuration  
+Build -> Configuration Manager...
 
 		Active solution configuration: Release
 
-5. Build Cmdlet  
-    Build -> Build AudioDeviceCmdlets
+6. Build Cmdlet  
+Build -> Build Solution
+
+		AudioDeviceCmdlets\SOURCE\bin\Release\AudioDeviceCmdlets.dll
 
-        AudioDeviceCmdlets\bin\Release\AudioDeviceCmdlets.dll
+7. Import Cmdlet to Windows PowerShell
```

Comment:
> > I recommend changing 'Windows PowerShell' to 'PowerShell on Windows'. Ever since they dropped the 'Core' suffix from PowerShell 6+, they've been saying 'Windows PowerShell' to refer to PowerShell 5.1 specifically.
> 
> Thank you @ImportTaste I did not know that. I'll try to read up a bit more on this.

An example of this is here. While the title of the article specifically states "Windows PowerShell 5.1", the article text goes on to use "Windows PowerShell" as shorthand for 5.1: https://docs.microsoft.com/en-us/powershell/scripting/whats-new/differences-from-windows-powershell?view=powershell-7.2 This is most notable in the "Product terminology used in the documentation" section.

Furthermore, $profile for 5.1 is typically in `$env:USERPROFILE\Documents\WindowsPowerShell`, while in 6+ it's typically located in `$env:USERPROFILE\Documents\PowerShell`

Using the phrase "PowerShell on Windows" should be enough of a disambiguation.

Reply:
> Using the phrase "PowerShell on Windows" should be enough of a disambiguation.

@ImportTaste I think I understand your intent a little better.  


> PowerShell can load many traditional Windows PowerShell modules without modification. Additionally, PowerShell 7 includes a Windows PowerShell Compatibility feature that allows you to use Windows PowerShell modules that still require the full framework.

If I understand correctly, this section of the article leads me to the same conclusion as you. Because even if at first I do want to imply **Windows PowerShell**, the fact remains that **PowerShell** on Windows can also load **Windows PowerShell** modules. This means that the **Windows PowerShell** vs. **PowerShell** distinction is irrelevant. What is relevant is the fact that it is on Windows.

Ok so that was a bit complicated at first, but I think I got the semantics down now.

Reply:
Yeah, you got it!